.github/workflows/ci.yml

```diff
@@ -26,20 +46,95 @@ jobs:
           sudo apt-get -y update
           sudo apt-get -y install protobuf-compiler libprotobuf-dev
 
+      - name: Run cargo check
+        run: cargo check --locked --workspace
+
+  check-wasm:
+    name: Check Wasm
+    runs-on: ubuntu-20.04
```

Comment:
Yes but that would be overkill. It only takes `1m 25s` currently so it takes less time than the longest running job. Since the jobs run in parallel anyway, improving it doesn't improve the total running time.